.github/workflows/deploy-backend.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy backend
+on:
+  push:
+    paths:
+      supabase/migrations/**.sql
```

Comment:
I wonder about the name of this workflow - it responds to changes in migrations but not other potential backend deployments, like deploying Edge Functions on changes in `supabase/functions`. I'm pretty sure that I had to [deploy Function](https://supabase.com/docs/guides/functions/deploy#deploy-your-edge-functions) changes manually, but I forgot to document in the README, sorry.

So I guess I'm saying: Should we change the name of this workflow to be about migrations, and raise a ticket for adding a workflow to deploy functions?

Reply:
Btw it's great to see that you're adding this, thank you!

Reply:
Ooh yeah that makes sense, I've made this specific to migrations now and raised a [ticket for function deployment](https://softwiretech.atlassian.net/browse/VFB-109!